src/test/helper.ts

```diff
@@ -30,8 +30,6 @@ async function activated() {
 			} else {
 				console.log('hashicorp.terraform is already active');
 			}
-			// make sure language server download is complete
-			await ext.exports.pathToBinary();
```

Comment:
Just to persist context for the future - why is this being removed?

Reply:
It was no longer doing what it claimed to -- setting the path is now separate from installing the language server.